Sources/schema-gen/PropertyDecl.swift

```diff
@@ -116,7 +117,7 @@ struct PropertyDecl {
       return ParsedBinding(
         patternBinding: binding,
         identifier: identifierPattern.identifier.text,
-        type: binding.typeAnnotation?.type.withoutTrivia().description
```

Comment:
@stackotter They had removed `withoutTrivia` entirely, perhaps I could try the following?
```swift
binding.typeAnnotation?.type.trimmedDescription
```

The doc on `trimmedDescription` says - **_The description of this node with leading whitespace of the first token and trailing whitespace of the last token removed_**.

Reply:
Ah yeah that sounds right. Can you please add that everywhere that `withoutTrivia` had to get removed? (I think there were more than just the one that I commented on)

Weird that they removed `withoutTrivia`, I guess since it was still all considered unstable they weren't scared of making breaking changes to improve the swift-syntax API before it got a bit more set in stone.